storage/api-docs/api-docs.json

```diff
@@ -18,7 +18,7 @@
             "description": "Production Server"
         },
         {
-            "url": "http://localhost:8000/api/v1",
+            "url": "http://localhost:8000/api/v1/api/v1",
```

Comment:
I don't get it why this happened?

Reply:
This is resolved... but I've created another issue (see below). :c

//cc @HerrLevin